Jenkinsfile.build

```diff
@@ -0,0 +1,32 @@
+pipeline {
+    agent {
+        label 'buildagent'
+    }
+    environment {
+        SBT_HOME = tool name: 'sbt-1.2.6', type: 'org.jvnet.hudson.plugins.SbtPluginBuilder$SbtInstallation'
+        SBT_OPTS = '-Xmx2g -XX:ReservedCodeCacheSize=128m -XX:+CMSClassUnloadingEnabled'
+        PATH = "${env.SBT_HOME}/bin:${env.PATH}"
+    }
+    stages {
+        stage('Compile') {
+            steps {   
+                sh 'git fetch --tags'
+                sh 'find ~/.sbt/1.0/staging/*/waves -type d -name target | xargs -I{} rm -rf {}'
+                sh 'sbt cleanAll'
+                sh 'sbt compile'
```

Comment:
You run sbt twice here, its way to long
Use ```sh 'sbt "cleanAll;compile"'``` instead

Reply:
done